pkg/infra/teams/team3/loot_strat.go

```diff
@@ -60,10 +61,12 @@ func (a *AgentThree) LootAction(
 	return proposedLoot
 }
 
+// this is really poor from infra - i'd just ignore tbh
 func (a *AgentThree) HandleLootInformation(m message.TaggedInformMessage[message.LootInform], baseAgent agent.BaseAgent) {
 	// submit a proposal to the leader
+	fmt.Println("Made it here")
 	switch m.Message().(type) {
-	case *message.StartLoot:
+	case message.LootInform:
 		// Send Proposal?
 		sendProposal := rand.Intn(100)
 		if sendProposal < a.personality {
```

Comment:
This function was never being called, which in turn meant that no proposals were ever being sent. I've added a driver function (below) in `RequestLootProposal`